mv_reports.sh

```
@@ -0,0 +1,54 @@
+# Build rsync command
+RSYNC_OPTS="-avu --include='*.xlsx' --include='*/' --exclude='*'"
+
+# Add dry-run if specified
+if [ "$DRY_RUN" = "--dry-run" ]; then
+    RSYNC_OPTS="$RSYNC_OPTS --dry-run"
+    echo "$(date): DRYRUN mode enabled" | tee -a "$LOG_FILE"
+fi
+
+# Execute rsync and log output
+echo "$(date): Starting transfer of .xlsx files from $SOURCE_DIR to $DEST_DIR" | tee -a "$LOG_FILE"
+rsync $RSYNC_OPTS "$SOURCE_DIR/" "$DEST_DIR/" 2>&1 | sed 's/^/'"$(date): "'/g' | tee -a "$LOG_FILE"
```

Shellcheck (0.10.0)

[warning] Quotes/backslashes in this variable will not be respected.

(SC2090)